src/tests/RegistrationTest.java

```diff
@@ -0,0 +1,9 @@
+package tests;
+
+public class RegistrationTest {
+
+	
+	public void createUser(){
```

Comment:
Hi Sonali,
Please add @Test annotation 
Where is assertion statement 

Thanks,
Prashant